spatial-tracking-explainer.md

```diff
@@ -461,7 +432,6 @@ interface XRPose {
 
 enum XRReferenceSpaceType {
   "identity",
```

Comment:
`viewer`

Reply:
Boy, I'm just failing at this search and replace thing left and right...